Signal/src/views/ContactCell.swift

```diff
@@ -39,53 +63,55 @@ class ContactCell: UITableViewCell {
     }
```

Comment:
Can you please attach screenshots with dynamic text size on minimum, default and maximum?